lib/database/attendance.ts

```diff
@@ -0,0 +1,125 @@
+import { client } from "../db";
+import { array, TypeOf, Any } from "io-ts";
+import { Attendance, AttendanceSchema, SingleUserAttendance, SingleUserAttendanceSchema, CreateAttendance, CreateAttendanceSchema, createAttendanceArrayType} from "../../models/attendance";
+import { decode } from "io-ts-promise";
+
+const AttendanceArraySchema = array(AttendanceSchema);
+type attendanceArrayType = TypeOf<typeof AttendanceArraySchema>;
```

Comment:
we shouldn't need to define this as a type, any time you use this, it should just be `Attendance[]`